app/scripts/modules/cloudfoundry/src/pipeline/stages/createServiceKey/cloudfoundryCreateServiceKeyStage.module.ts

```diff
@@ -9,7 +9,7 @@ Registry.pipeline.registerStage({
   component: CloudfoundryCreateServiceKeyStageConfig,
   controller: 'BaseProviderStageCtrl as baseProviderStageCtrl',
```

Comment:
Can we strike `controller` since this is a React stage?

Reply:
👍